Task6/src/main/java/game/view/View.java

```diff
@@ -0,0 +1,30 @@
+package game.view;
+
+
+import game.model.Door;
+import game.model.Room;
+
+import java.util.List;
+
+public class View {
+    Room room =new Room();
+
+    public void printDoors(){
+        System.out.println("Двері і їхній вміст: ");
+        System.out.println(room.describeDoors());
+    }
+
+    public void printDeadDoorsCount(){
+        System.out.println("Кількість дверей, де можуть вбити героя: " + room.countDeadlyDoors());
+    }
+
+    public void printSafeDoorOrder(){
+        if (!room.getSafeDoorOrder().isEmpty()) {
```

Comment:
In LOC(Line Of Code) 22 and 23 you invoke getSafeDoorOrder twice. It is unnecessary. Create field to store method result after first use after LOC 21.
Result of current code method invoked twice: 
![image](https://github.com/olesiatsikhovych/JavaTasks/assets/50113284/820f5cdc-0836-44b3-8b4e-edb7514dee56)

Reply:
Plus screenshot is great example of log usefulness

Reply:
done